<launch>

<node name="balloon_estimation" pkg="balloon_estimation" type="balloon_estimation_node" output="screen">

  <param name="bounding_box_topic" type="string" value="/target_bbx" />
  <param name="uav_number" type="int" value="10" />
  <param name="vel_noise" type="double" value="0.01" />
  <param name="meas_noise_multiplyer" type="double" value="1.0" /> 
  <param name="meas_noise" type="double" value="0.3" />
  
  <rosparam param="cam_k">[382.0604,382.2626,322.2945,244.5334]</rosparam>
  <rosparam param="cam_d">[-0.0503,0.0367,0,0]</rosparam>

  <!-- <rosparam param="T_CtoI">
            [
             1, 0,  0,  0,
             0,-1,  0, -0.03567,
             0, 0, -1, -0.14677,
             0, 0,  0,  1.0000
            ]
  </rosparam>
  <rosparam param="T_GPStoI">
            [
             1, 0,  0,  0,
             0, 1,  0,  0.107,
             0, 0,  1,  0,
             0, 0,  0,  1.0000
            ]
  </rosparam> -->
   <rosparam param="T_CtoI">
            [
             0, 0,  1,   0.14677,
             -1, 0,  0,  0,
             0, -1, 0, -0.03567,
             0, 0,  0,  1.0000
            ]
  </rosparam>
  <rosparam param="T_GPStoI">
            [
             1, 0,  0,  0,
             0, 1,  0,  0,
             0, 0,  1,  0.107,
             0, 0,  0,  1.0000
            ]
  </rosparam>

</node> 
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find balloon_estimation)/launch/display.rviz" />                                                                                                                                                                                                                                                                         

</launch>
